packages/salesforcedx-vscode-core/src/commands/forceOpenDocumentation.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+export const auraDocUrl =
+  'https://developer.salesforce.com/tools/vscode/en/aura/writing';
+export const apexDocUrl =
+  'https://developer.salesforce.com/tools/vscode/en/apex/writing';
+export const soqlDocUrl =
+  'https://developer.salesforce.com/tools/vscode/en/soql/soql-builder';
+export const lwcDocUrl =
+  'https://developer.salesforce.com/tools/vscode/en/lwc/writing';
+export const functionsDocUrl =
+  'https://developer.salesforce.com/tools/vscode/en/functions/overview';
+export const defaultDocUrl = 'https://developer.salesforce.com/tools/vscode';
+
+const auraPath = '/force-app/main/default/aura/';
+const apexClassesPath = '/force-app/main/default/classes/';
+const lwcPath = '/force-app/main/default/lwc/';
+const functionsPath = '/functions/';
+
+const apexExtension = '.apex';
+const soqlExtension = '.soql';
```

Comment:
The const above seem like they are reusable outside this command (esp the path/extension values).  Think we could move these all to `packages/salesforcedx-utils-vscode/src/constants.ts`?

Reply:
It turns out there are 11 files named, "constants.ts". There doesn't seem to be any occurrences of constants being using from other packages (like utils) so I placed the extension constants in packages/salesforcedx-vscode-core/src/constants.ts,   Let me know if you think they should be in a different file.

Reply:
👍 that works.  we can figure out the right place for all the constants later.